source/common/http/utility.cc

```diff
@@ -186,6 +186,125 @@ bool Utility::isWebSocketUpgradeRequest(const HeaderMap& headers) {
                     Http::Headers::get().UpgradeValues.WebSocket.c_str())));
 }
 
+bool Utility::hasGrpcContentType(const HeaderMap& headers) {
```

Comment:
Why was this moved? It seems to conceptually be part of the gRPC support in Envoy. It makes it a bit hard to see what has changed due to GH limitations.

Reply:
In an intermediate version I had `Http::Utility::SendLocalReply()` needing to call this, but now it's not needed any more. I'll move this and `isGrpcResponseHeader()` back, thanks!